apps/explorer/lib/explorer/exchange_rates/source/mobula.ex

```diff
@@ -4,6 +4,7 @@ defmodule Explorer.ExchangeRates.Source.Mobula do
   """
 
   require Logger
+  alias Explorer.Chain
```

Comment:
Let's rewrite this as follows 


```suggestion
  alias Explorer.Chain.Hash.Address
```